### ISSUE TITLE
Accessibility Check

### ISSUE THREAD

**Issue by hiimchrislim** [utmgdsc/PollVotingSystem#58]
**Describe the solution you'd like**

It would be good to run through an accessibility check and take note of what we're missing/could be changed to improve our overall website accessibility.
